packages/@react-spectrum/illustrated-message/stories/IllustratedMessage.stories.tsx

```diff
@@ -10,15 +10,17 @@
  * governing permissions and limitations under the License.
  */
 
+import {Content, Header, SVGIllustration} from '@react-spectrum/view';
+import {Heading} from '@react-spectrum/typography';
 import {IllustratedMessage} from '../';
 import React from 'react';
 import {storiesOf} from '@storybook/react';
 
 function noResultsImg() {
   return (
-    <svg width="150" height="103" viewBox="0 0 150 103">
+    <SVGIllustration width="150px" height="103px" viewBox="0 0 150 103">
```

Comment:
SVGIllustration should take an svg, you should be able to base it off of Icon and UIIcon